src/vdf_io/export_vdf/turbopuffer_export.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import turbopuffer as tpuf
+from vdf_io.names import DBNames
+from vdf_io.util import clean_documents
+
+
+class ExportTurbopuffer:
```

Comment:
it should inherit from ExportVDB and implement the required functions based on that (like get_data()).